extensions/amp-story/1.0/_locales/en.js

```diff
@@ -165,6 +165,11 @@ export default /** @const {!LocalizedStringBundleDef} */ ({
     description: 'Label for the expandable share widget shown in the ' +
         'desktop UI.',
   },
+  [LocalizedStringId.AMP_STORY_TOOLTIP_OPEN_IN_TWITTER]: {
+    string: 'View on Twitter',
```

Comment:
Actually, I'm not sure if this is the best string?  This makes me feel like when I click on the tooltip, it will actually navigate to the tweet on twitter.com.

/cc @hongwei1990

Reply:
changed it, let me know what you think :)